tests/test_Plant.py

```diff
@@ -0,0 +1,59 @@
+""" 
+    Tests the file Plant.py
+    Focuses on
+        -objects having the right type
+        -functions returning the correct value
+        -functions setting the correct value
+"""
+from src.Plant import Plant
+import datetime
+
+
+def test_types():
+    testing = Plant()
+    testing.setName("")
+    testing.setLast_watered(datetime.datetime.now())
+    testing.setWatering_interval(datetime.timedelta(0))
+    testing.setWatering_time(datetime.datetime.now().time())
+    assert type(testing.getName()) is str
+    assert type(testing.getLast_watered()) is datetime.datetime
+    assert type(testing.getWatering_interval()) is datetime.timedelta
+    assert type(testing.getWatering_time()) is datetime.time
+
+
+def test_getting():
+    parameters = (datetime.timedelta(2),datetime.datetime.now())
```

Comment:
Any particular reason `parameters` isn't an object?

Also, have a look at https://pypi.org/project/freezegun/ for handling `datetime.datetime.now()`. Generally speaking, you don't want to use the real time in tests and you usually want control over when "now" is. While using the real now will work for the moment, you'll run into problems soon as you try to handle the actual watering day logic checks, especially when combined with a database, and other time-based functions. So this is a good opportunity to get familiar with mocking in general and faking time in particular.

Reply:
At the time of writing the actual code, making the parameters separate seemed important, but now it just seems unnecessary. Also thanks for the tip about datetime.now, it till save a lot of head scratching in the future.